fhir-server-test/src/test/java/com/ibm/fhir/server/test/BundleTest.java

```diff
@@ -2331,6 +2334,94 @@ public void testTransactionConditionalDeletes() throws Exception {
         assertGoodGetResponse(responseBundle.getEntry().get(1), Status.OK.getStatusCode(), HTTPReturnPreference.MINIMAL);
     }
 
+    /**
+     * Test transaction bundle put with post local reference dependency.
```

Comment:
Update testcase description to match what is being tested.

Reply:
```suggestion
     * Sets UPDATE_IF_MODIFIED_HEADER_NAME  and posts a transaction bundle with an update and a patch; both should be skipped on the server
```